peripherals/tim/tim16.yaml

```diff
@@ -0,0 +1,42 @@
+# TIM16 and TIM17 peripherals
+# All devices that has TIM16 has TIM17 except stm32l4x{1,2,3} where only TIM16
+# exists.
+#
+
+"TIM16":
+  CR1:
+    OPM:
+      NotStopped: [0, "Not stopped at update event"]
+      Stopped: [1, "Counter stops counting at next update event"]
+    ARPE:
+      NotBuffered: [0, "TIMx_ARR register is not buffered"]
+      Buffered: [1, "TIMx_ARR register is buffered"]
+  CR2:
+    CCPC:
+      NotPreloaded: [0, "CCxE, CCxNE and OCxM bits are not preloaded"]
+      Preloaded: [1, "CCxE, CCxNE and OCxM bits are preloaded"]
+    CCUS:
+      WithoutRisingEdge: [0, "Capture/compare are updated only by setting the COMG bit"]
+      WithRisingEdge: [1, "Capture/compare are updated by setting the COMG bit or when an rising edge occurs on TRGI"] 
```

Comment:
Is WithoutRisigngEdge good or something like Default better?

Reply:
I don't mind either way here, your choice.